infrastructure.tf

```diff
@@ -619,61 +732,61 @@ resource oci_dns_zone openshift {
   zone_type      = "PRIMARY"
   depends_on     = [oci_core_subnet.private]
 }
-
+ 
 resource oci_dns_rrset openshift_api {
   domain = "api.${var.cluster_name}.${var.zone_dns}"
   items {
     domain = "api.${var.cluster_name}.${var.zone_dns}"
     rdata  = var.enable_private_dns ? local.lb_private_addr : local.lb_public_addr
     rtype  = "A"
-    ttl    = "30"
+    ttl    = "3600"
   }
   rtype           = "A"
   zone_name_or_id = oci_dns_zone.openshift.id
 }
-
+ 
 resource oci_dns_rrset openshift_apps {
   domain = "*.apps.${var.cluster_name}.${var.zone_dns}"
   items {
     domain = "*.apps.${var.cluster_name}.${var.zone_dns}"
     rdata  = var.enable_private_dns ? local.lb_private_addr : local.lb_public_addr
     rtype  = "A"
-    ttl    = "30"
+    ttl    = "3600"
   }
   rtype           = "A"
   zone_name_or_id = oci_dns_zone.openshift.id
 }
-
+ 
 resource oci_dns_rrset openshift_api_int {
   domain = "api-int.${var.cluster_name}.${var.zone_dns}"
   items {
     domain = "api-int.${var.cluster_name}.${var.zone_dns}"
-    rdata  = var.enable_private_dns ? local.lb_private_addr : local.lb_public_addr
+    rdata  = local.lb_private_addr
     rtype  = "A"
-    ttl    = "30"
+    ttl    = "3600"
   }
   rtype           = "A"
   zone_name_or_id = oci_dns_zone.openshift.id
 }
-
+ 
 resource "time_sleep" "wait_180_seconds" {
   depends_on = [oci_core_vcn.openshift_vcn]
   create_duration = "180s"
 }
-
+ 
 data "oci_core_vcn_dns_resolver_association" "dns_resolver_association" {
   vcn_id     = oci_core_vcn.openshift_vcn.id
   depends_on = [time_sleep.wait_180_seconds]
 }
-
+ 
 data "oci_dns_resolver" "dns_resolver" {
   depends_on = [
     data.oci_core_vcn_dns_resolver_association.dns_resolver_association
   ]
   resolver_id = data.oci_core_vcn_dns_resolver_association.dns_resolver_association.dns_resolver_id
   scope       = "PRIVATE"
 }
-
+ 
 resource oci_core_instance_configuration master_node_config {
   count          = local.create_openshift_instance_pools ? 1 : 0
```

Comment:
We need to consider a case for multiAd regions like Ashburn, Phoenix and more

Reply:
We will investigate and most likely address this in the next sprint